build.sbt

```diff
@@ -0,0 +1,36 @@
+name := "untitled"
+
+version := "0.1"
+
+scalaVersion := "2.12.9"
+
+val akkaHttpVersion = "10.1.9"
+val slickVersion = "3.3.2"
+val akkaVersion = "2.5.25"
+val scalatestVersion = "3.0.8"
+val postgresVersion = "42.2.6"
+val unicornVersion = "1.3.3"
```

Comment:
unused... but: why did you want to use it at the beginning? And why you abandoned that idea in the end?